OpenRA.Game/Settings.cs

```diff
@@ -269,6 +269,12 @@ public class KeySettings
 		public Hotkey SupportPower05Key = new Hotkey(Keycode.UNKNOWN, Modifiers.None);
 		public Hotkey SupportPower06Key = new Hotkey(Keycode.UNKNOWN, Modifiers.None);
 
+		public Hotkey PauseAltKey = new Hotkey(Keycode.SPACE, Modifiers.None);
```

Comment:
I suggest just using the existing pause key, since that doesn't do anything in replays anyway.